mautic/mautic
# Fix dashboard query performance

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          |no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 


#### Description:
Dashboard stat query takes abnormally long time to load or is out of date.

Note:
There are multiple formats used to identify date value format used for ordering and adding empty values

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check dashboard page with widget `Upcoming emails` enabled and you'll get same result as before applying this PR
3. Other widgets are providing the same information as before applying this PR
